src/encoded/static/components/facets/defaults.js

```diff
@@ -0,0 +1,572 @@
+    // parameters of this type check the "either" button.
+    let currentOption = 'either';
+    if (relevantFilters.length === 1) {
+        if (relevantFilters[0].term === '1' || relevantFilters[0].term === 'true') {
+            currentOption = 'true';
+        } else if (relevantFilters[0].term === '0' || relevantFilters[0].term === 'false') {
+            currentOption = 'false';
+        }
+    }
+
+    // We have to build the new query string unless the user clicked the "either" radio button,
+    // which uses the `remove` link from the relevant filter. This callback gets memoized to avoid
+    // needlessly rerendering this component, and its dependencies should normally not change until
+    // the user clicks a term.
+    const handleRadioClick = React.useCallback((event) => {
+        const { value } = event.target;
+        let href;
+        if (value === 'either') {
+            // If the user can check the "either" button then we know the query string has only one
+            // of these "exists" elements, so just get the first relevant filter's `remove`
+            // property.
+            href = url.parse(relevantFilters[0].remove).search || relevantFilters[0].remove;
+        } else {
+            // User clicked the "true" or "false" radio buttons. Replace any existing relevant query
+            // element with one corresponding to the clicked radio button.
+            const clickedElementName = event.target.getAttribute('name');
+            const query = new QueryString(queryString);
+            query.replaceKeyValue(clickedElementName, value);
+            href = `?${query.format()}`;
+        }
+        reactContext.navigate(href);
+    }, [relevantFilters, queryString, reactContext]);
+
+    return (
+        <fieldset className="facet">
+            <legend>{facet.title}</legend>
+            <div className="facet__content--exists">
+                {facet.terms.map(term => (
+                    <div key={term.key_as_string} className="facet__radio">
+                        <input type="radio" name={facet.field} value={term.key_as_string} id={term.key_as_string} checked={currentOption === term.key_as_string} onChange={handleRadioClick} />
+                        <label htmlFor={term.key}>
+                            <div className="facet__radio-label">{term.key_as_string}</div>
+                            <div className="facet__radio-count">{term.doc_count}</div>
+                        </label>
+                    </div>
+                ))}
+                <div className="facet__radio">
+                    <input type="radio" name={facet.field} value="either" id={`${facet.field}-either`} checked={currentOption === 'either'} onChange={handleRadioClick} />
+                    <label htmlFor={`${facet.field}-either`}>
+                        <div className="facet__radio-label">either</div>
+                        <div className="facet__radio-count">{facet.total}</div>
```

Comment:
I would change this to "Either" and convert `term.key_as_string` to be uppercase, but this is a style preference and I'm not sure what the Encode style should be.

Reply:
I’ll leave it lowercase for now, only because "exists" facets have always been lowercase, and I haven’t heard otherwise from the wrangling side yet.